pages/linux/tshark.md

```diff
@@ -0,0 +1,31 @@
+# tshark
+
+> Packet analysis tool, cLI version of wireshark.
+
+- Monitor everything on localhost:
+
+`tshark`
+
+- Filter captured packets by method:
+
+`tshark -Y 'http.request.method == "GET"'`
+
+- Decode the current port with a specific protocol (ex: http):
+
+`tshark -d tcp.port==8888,http`
+
+- Format of captured output:
+
+`tshark -T json`
+
+- Display fields to output:
+
+`tshark -T fields -e http.request.method -e http.request.uri -e ip.dst`
+
+- Write captured packet to a file:
+
+`tshark -w -> {{file_name}}.pcap`
```

Comment:
```suggestion
`tshark -w {{file_name}}.pcap`
```

Why bother with pipes? The `-w` option already takes an output file name.

Reply:
there are 2 ways
`tshark -w {{path/to/file}}` or `tshark -w - > {{path/to/file}}` the difference is that `-w -` is putting everything to stdout.

```suggestion
`tshark -w -> {{path/to/file}}`
```